pkgs/development/libraries/qt-6/default.nix

```diff
@@ -69,47 +69,49 @@ let
         ];
       };
       env = callPackage ./qt-env.nix { };
-      full = callPackage ({ env, qtbase }: env "qt-full-${qtbase.version}"
-      # `with self` is ok to use here because having these spliced is unnecessary
-      ( with self;[
-        qt3d
-        qt5compat
-        qtcharts
-        qtconnectivity
-        qtdatavis3d
-        qtdeclarative
-        qtdoc
-        qtgraphs
-        qtgrpc
-        qthttpserver
-        qtimageformats
-        qtlanguageserver
-        qtlocation
-        qtlottie
-        qtmultimedia
-        qtmqtt
-        qtnetworkauth
-        qtpositioning
-        qtsensors
-        qtserialbus
-        qtserialport
-        qtshadertools
-        qtspeech
-        qtquick3d
-        qtquick3dphysics
-        qtquickeffectmaker
-        qtquicktimeline
-        qtremoteobjects
-        qtsvg
-        qtscxml
-        qttools
-        qttranslations
-        qtvirtualkeyboard
-        qtwebchannel
-        qtwebengine
-        qtwebsockets
-        qtwebview
-      ] ++ lib.optionals (!stdenv.isDarwin) [ qtwayland libglvnd ])) { };
+      full = callPackage
```

Comment:
Please drop the formatting changes.

Reply:
Please reconsider this one. Commits have been separated between the formatting and adding mingw support. This is what I have done for every merge request mentioned in these issues
https://github.com/NixOS/nixpkgs/issues/274274
https://github.com/NixOS/nixpkgs/issues/272538

Reply:
Fine then.

Reply:
It is a bug in nixpkgs-fmt that the after the callPackage a line break is added. Also it makes the entire PR looking bigger than it is.